libraries/bsg_manycore.cpp

```diff
@@ -524,51 +167,34 @@ int  hb_mc_manycore_init(hb_mc_manycore_t *mc, const char *name, hb_mc_manycore_
                 return r;
         }
 
-        // initialize private data
-        if ((err = hb_mc_manycore_init_private_data(mc)) != HB_MC_SUCCESS)
-                goto cleanup;
```

Comment:
The gotos were done incorrectly. It's more verbose, but I switched to if-statements